app/models/api_docs/service.rb

```diff
@@ -38,6 +44,34 @@ class ApiDocs::Service < ApplicationRecord
 
   validates :base_path, non_localhost: true
 
+  def owner_account
+    return unless owner
+
+    owner_type == 'Account' ? owner : owner.account
+  end
```

Comment:
It would be so nice if `Account#account` returned `self`:

```ruby
  delegate :account, to: :owner, prefix: true, allow_nil: true
```

...but it does NOT 😞.